Sources/PT.PM.Common/Nodes/Expressions/UnaryOperatorExpression.cs

```diff
@@ -36,7 +36,12 @@ public override string ToString()
             UnaryOperator op = Operator.UnaryOperator;
             if (UnaryOperatorLiteral.PrefixTextUnaryOperator.ContainsValue(op))
             {
-                return $"{Operator}{Expression}";
+                UnaryOperator unaryOp = Operator.UnaryOperator;
```

Comment:
you already have this variable, it is declared above (i mean `UnaryOperator op` on line 36)